src/types/index.ts

```diff
@@ -1,2 +1,3 @@
 export type Headings = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
 export type SortDirection = "none" | "ascending" | "descending";
+export type ValuesOf<T> = T[keyof T];
```

Comment:
Maybe `ValueOf ` because it's used to get any one of the values (same as `keyof` and `typeof`).